src/features/user/Settings/EditProfile/EditProfileForm.tsx

```diff
@@ -490,170 +589,235 @@ export default function EditProfileForm() {
+          </StyledInputLabel>
           <Controller
-            name="city"
-            control={control}
-            rules={{
-              required: t('validationErrors.cityRequired'),
-              pattern: {
-                value: /^[\p{L}\sß.,()-]+$/u,
-                message: t('validationErrors.cityInvalid'),
-              },
-            }}
-            render={({ field: { onChange, value, onBlur } }) => (
-              <TextField
-                label={t('fieldLabels.city')}
-                onChange={onChange}
-                onBlur={onBlur}
-                value={value}
-                error={errors.city !== undefined}
-                helperText={errors.city !== undefined && errors.city.message}
-              />
-            )}
-          />
-          <Controller
-            name="zipCode"
+            name="bio"
             control={control}
             rules={{
-              required: t('validationErrors.zipCodeRequired'),
-              pattern: {
-                value: postalRegex as RegExp,
-                message: t('validationErrors.zipCodeInvalid'),
+              maxLength: {
+                value: 300,
+                message: t('validationErrors.maxChars', { max: 300 }),
               },
             }}
             render={({ field: { onChange, value, onBlur } }) => (
               <TextField
-                label={t('fieldLabels.zipCode')}
                 onChange={onChange}
                 onBlur={onBlur}
                 value={value}
-                error={errors.zipCode !== undefined}
-                helperText={
-                  errors.zipCode !== undefined && errors.zipCode.message
-                }
+                multiline
+                rows={4}
+                error={errors.bio !== undefined}
+                helperText={errors.bio !== undefined && errors.bio.message}
               />
             )}
           />
-        </InlineFormDisplayGroup>
-        <AutoCompleteCountry
-          defaultValue={country}
-          onChange={setCountry}
-          label={t('fieldLabels.country')}
-          name="editProfile"
-          countries={allCountries}
-        />
-        <InlineFormDisplayGroup type="other">
-          <div>
+        </EditProfileInputContainer>
+
+        <div className={styles.customiseProfileToggles}>
+          <h2>{t('customiseProfileFields.customiseProfileTitle')}</h2>
+          <InlineFormDisplayGroup type="other">
+            <div>
+              <label
+                htmlFor="is-public"
+                className={styles.customiseProfileToggleLabel}
+                style={{ cursor: 'pointer' }}
+              >
+                {t('customiseProfileFields.switchVisibility')}
+              </label>
+              <br />
+              <label className={styles.isPrivateAccountText}>
+                {t('customiseProfileFields.publicProfileText')}
+              </label>
+            </div>
```

Comment:
Reconsider this structure

1. I think multiple label tags are not advisable for a single field/control.
2. publicProfileText is a simple field description, and this structure could easily be reused in other places. So giving a className like isPrivateAccountText makes identifying that opportunity difficult. I'd suggest generalizing it.